doc/rados/operations/health-checks.rst

```diff
@@ -1157,6 +1157,26 @@ The location of an OSD can be found with::
 
   ceph osd find osd.<id>
 
+PENDING_CREATING_PGS
+____________________
+
+One or more OSD have pending PGs creation due to various reasons, like
+incorrect crush map, dead OSDs in crush map etc. Additional message
+"(PGs creation pending, temporarily increase 'osd_max_pg_per_osd_hard_ratio')"
+in the pending PGs creation health message denotes one or more OSD have withheld
+PGs creation due to overdose protection meaning the number of PGs on OSD
+is more or equal to `mon_max_pg_per_osd * osd_max_pg_per_osd_hard_ratio` value.
```

Comment:
Suggest slight changes:

One or more OSDs have PGs pending creation for reasons that include a
flawed CRUSH map or rules and dead OSD. The additional text
"(PGs creation pending, temporarily increase 'osd_max_pg_per_osd_hard_ratio')"
in the pending PG creation health message indicates that one or more OSDs have blocked
PG creation due to overdose protection.  This means that the number of PGs on an OSD
is greather than or equal to `mon_max_pg_per_osd * osd_max_pg_per_osd_hard_ratio`.